.circleci/config.yml

```diff
@@ -160,9 +160,9 @@ commands:
       - run:
           name: "Publish to cloudflare pages (staging)"
           command: |
-            npm i wrangler@2.0.19
+            npm i wrangler@3.1.0
             cd packages/core
-            npx wrangler pages publish dist/ --project-name=deriv-app-pages --branch=staging
+            npx wrangler pages deploy dist/ --project-name=deriv-app-pages --branch=staging
```

Comment:
`publish` is deprecated and will be removed in `wrangler v4`. It'll be replaced by `deploy` according to [this](https://developers.cloudflare.com/workers/wrangler/deprecations/) documentation